src/SCardCall.hpp

```diff
@@ -48,45 +48,39 @@ void SCardCall(const char* callerFunctionName, const char* file, int line,
 {
     // TODO: Add logging - or is exception error message enough?
 
-#ifdef __APPLE__
-    // Apple version of pcsc-lite uses unsigned code constants
-    const uint32_t result = scardFunction(args...);
-#else
-    // Standard version of pcsc-lite uses signed code constants
     const LONG result = scardFunction(args...);
-#endif
 
     // TODO: Add more cases when needed.
     switch (result) {
     case SCARD_S_SUCCESS:
         return;
-    case SCARD_E_NO_SERVICE:
```

Comment:
These casts look redundant as pcsc-lite already defines those macros as long:
```
/usr/local/include/PCSC/pcsclite.h
#define SCARD_E_NO_SERVICE           ((LONG)0x8010001D) /**< The Smart card resource manager is not running. */
```

Reply:
This might be true, but other OS-es might not have these casts (eg. magOS)